src/node/blockstorage.h

```diff
@@ -138,7 +138,17 @@ class BlockManager
      */
     std::unordered_map<std::string, PruneLockInfo> m_prune_locks GUARDED_BY(::cs_main);
 
+    const kernel::BlockManagerOpts m_opts;
+
 public:
+    using Options = kernel::BlockManagerOpts;
+
+    explicit BlockManager(Options opts)
+        : m_prune_mode{opts.prune_target > 0},
+          m_opts{std::move(opts)} {};
+
+    std::atomic<bool> m_importing{false};
```

Comment:
nit: I think it would be nice if access to this was wrapped in a getter/setter, e.g. `Importing(), {Start,Stop}Importing()`.

My thinking is that we might (in the future) want to mock the `BlockManager` to avoid expensive i/o in the fuzz tests for example. Having a clear interface defined would make that easier. A simple member like this is not a huge blocker for that so feel free to ignore.